docker-compose.yml

```diff
@@ -17,7 +17,7 @@ x-aliases:
         - agent
         - ddagent_integration
         - request-replayer
-        - elasticsearch2_integration
+        #- elasticsearch2_integration
```

Comment:
Not sure about this, but there's no elasticsearch2 container for arm... Leaving this here will just completely block the containers from running on arm, so I think we can leave them commented out?

Reply:
For now, this seems okay.